apostrophecms/apostrophe
# Pro 6094 shortcut display

## Summary

* Update conflicted shortcuts
* Add shortcuts to additional modals

## What are the specific steps to test this change?

1. Use `G` then `Shift+I` shortcut to open the Image Tags manager modal.
2. Use `G` then `Shift+F` shortcut to open the File Tags manager modal.
3. Detect shortcut conflicts when using multiple shortcuts.
4. Use `T` then `S` shortcut to open the Personal Settings.
5. Use `T` then `D` shortcut to open the Submitted Drafts.
6. Check the scrollbar on the shortcut list.

## What kind of change does this PR introduce?
*(Check at least one)*


## Make sure the PR fulfills these requirements:


If adding a new feature without an already open issue, it's best to open a **feature request issue** first and wait for approval before working on it.

**Other information:**
